### PR TITLE
Minor refactoring in docker build scripts

### PR DESCRIPTION
**Pull request by daxian-dbw**
Some name changes, comment updates, and random fixes.

### CODE REVIEW

**Review by adityapatwardhan — APPROVED** [review]

File: vstsBuild/vstsBuild.psm1 [hunk 1]
```diff
@@ -80,31 +80,33 @@ function Invoke-VstsPublishBuildArtifact
...
                     Write-Host "##vso[task.setvariable variable=$Variable]$unzipPath"
-                    # Set a variable in the current process.  PowerShell will not pickup the variable until the process is restarted otherwise.
+                    # Set a variable in the current process. PowerShell will not pickup the variable until the process is restarted otherwise.
                     Set-Item env:\$Variable -Value $unzipPath
                 }
                 Expand-Archive -Path $fileName -DestinationPath $unzipPath
             }
 
-            Write-Host "##vso[artifact.upload containerfolder=results;artifactname=$leafFileName]$FileName"
+            Write-Host "##vso[artifact.upload containerfolder=results;artifactname=$leafFileName]$fileName"
```
Comment (TravisEz13) [inline vstsBuild/vstsBuild.psm1]: can we update the container folder to be `$Bucket`?

**Review by TravisEz13 — APPROVED** [review]

File: dockerBasedBuild/dockerBasedBuild.common.psm1 [hunk 2]
```diff
@@ -579,7 +578,7 @@ class BuildData
     [String]$BinaryBucket = 'release'
 
     # Optional: the number of expected binaries from the build.
-    [Int]$BinariesExpected = -1
+    [Int]$ArtifactsExpected = -1
```
Comment (TravisEz13) [inline dockerBasedBuild/dockerBasedBuild.common.psm1:581]: This requires JSON changes.  Make sure to update the VSCode repo

Comment (daxian-dbw) [reply dockerBasedBuild/dockerBasedBuild.common.psm1:581]: The `build.json` file in `vscode-powershell` repo doesn't contain the `BinariesExpected` key.

File: vstsBuild/vstsBuild.psm1 (hunk 1 above)
Comment (daxian-dbw) [reply vstsBuild/vstsBuild.psm1]: Updated. Use `$Bucket` instead.

**Comment by daxian-dbw** [comment]
@TravisEz13 Can you please take another look?

**Comment by TravisEz13** [comment]
This change looks good. let me look at the other PR.

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**